plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar310/TreeBuilder310.java

```diff
@@ -679,13 +680,17 @@ public final SimpleNode onCloseNode(SimpleNode n, int arity) throws Exception {
 
             case JJTATTR:
                 if (arity == 1) {
+                    if (isPeekedNodeWildcardPattern()) {
+                        stack.popNode();
+                        return getWildcardPattern();
```

Comment:
I think you don't need the `getWildcardPattern` (the only thing different from the `Name` is the `ctx`, so, you can just change it directly in the popped node and return it -- otherwise you'd need to copy the other attributes such as specials, etc.).